package.json

```diff
@@ -81,6 +81,8 @@
     "pg": "^8.9.0",
     "pg-hstore": "^2.3.4",
     "query-string": "^5.1.0",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
```

Comment:
@Anshgrover23 you can't add react here, it's the Node application, and react dependency it's on the front-end package.json

Reply:
ok i will remove them sorry for that mistake